nautobot/docs/development/getting-started.md

```diff
@@ -246,6 +246,9 @@ After opening the project directory in VS Code in a supported environment, you w
 
 To start Nautobot, select **Run Without Debugging** or **Start Debugging** from the Run menu. Once Nautobot has started, you will be prompted to open a browser to connect to Nautobot.
 
+!!! note
+    You can run tests with `nautobot-server test --config=nautobot/core/tests/nautobot_config.py` while inside the Container.
+
```

Comment:
Ideally the `invoke unittest` task should be updated to do the right thing. A fix that's exclusive to VS Code users (as the placement of this note in the document suggests?) is not ideal. :-)

Reply:
Invoke unittest already works, but when you are working inside the vscode devcontainer, it is nice to just run `nautbot-server test`.

Reply:
To clarify - `invoke unittest` shows this same issue if the worker container is running. I didn't mean that fixing this for vscode was the wrong thing to do, just that I'd also like it fixed for the invoke task as well. :)